pkg/broker/filter/filter_handler_test.go

```diff
@@ -32,8 +32,8 @@ import (
 	cehttp "github.com/cloudevents/sdk-go/pkg/cloudevents/transport/http"
 	"github.com/google/go-cmp/cmp"
 	"go.uber.org/zap"
-	"k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
revert this change? probably some formatting thing as well

Reply:
as discussed, keeping `metav1`